src/app/feedback/feedback-view.component.ts

```diff
@@ -111,15 +111,16 @@ export class FeedbackViewComponent implements OnInit, OnDestroy {
   }
 
   editTitle(mode) {
-    this.editTitleMode = mode;
+    this.couchService.put(this.dbName + '/' + this.feedback._rev, this.feedback).subscribe(
```

Comment:
The name of this function could be a little clearer, but this simply switches from just showing the title text or showing an input field to edit the title text.

This function should remain as is.